### ISSUE TITLE
in "[array addObject:@(app.processIdentifier)]" "@(" not handled

### ISSUE THREAD

**Issue by kjk** [drkameleon/iswift#35]
Trying to convert https://github.com/gnachman/iTerm2/blob/master/sources/AppleScriptTest.m

```
- (NSArray *)processIdsForTestApp {
    NSMutableArray *array = [NSMutableArray array];
    for (NSRunningApplication *app in [[NSWorkspace sharedWorkspace] runningApplications]) {
        if ([app.bundleIdentifier isEqualToString:kTestBundleId]) {
            [array addObject:@(app.processIdentifier)];
        }
    }
    return array;
}
```
fails with "Unexpected (" in " [array addObject:@(app.processIdentifier)];" line.

**Comment by drkameleon** [comment]
True, as well. (And thanks for reminding me...).

- `@"..."` string *are* recognized
- `@[...]" arrays too
- `@{...}" dictionaries too

I guess it's the only one missing - so let's see what can be done!

**Comment by drkameleon** [comment]
Fixed as of the upcoming 1.5.
